src/utils/generateKeys.js

```diff
@@ -0,0 +1,7 @@
+import nacl from 'tweetnacl';
```

Comment:
@jstarry - heads up.  Should we move this into solana-web3.js?   Generating keypairs correctly is pretty important, I think we want this logic centralized

Reply:
@mvines i made it similar to https://github.com/solana-labs/example-webwallet/blob/f15144c5a7d84f3debd6fdf5909c8a2c35fa256d/src/store.js#L38

Reply:
@mvines agreed, created an issue: https://github.com/solana-labs/solana-web3.js/issues/487